image/preview/Dockerfile

```diff
@@ -12,14 +12,19 @@ ENV VERSION=${VERSION} \
     IMAGE_BINARY="/usr/bin/convert" \
     IMAGE_TYPE="png" \
     IMAGE_THUMBNAIL_COMMAND="@BINARY@ @INPUT@[0] -auto-orient -resize 225^ @OUTPUT@" \
-    IMAGE_PREVIEW_COMMAND="@BINARY@ @INPUT@[0] -auto-orient -resize 800x600 @OUTPUT@"
+    IMAGE_PREVIEW_COMMAND="@BINARY@ @INPUT@[0] -auto-orient -resize 800x600 @OUTPUT@" \
+    MAGICK_CONFIGURE_PATH='/extractor'
 
 WORKDIR /extractor
 
-RUN apk add --no-cache imagemagick
+RUN apt update && apt-get install build-essential && \
```

Comment:
any reason we install a custom version of imagemagick, this is broken since it tries to cd to the wrong folder name. Also this will result in a large docker image.

Reply:
Didn't mean to include this old build code, replaced with apt install imagemagick.